.github/workflows/build_docker_image.yml

```diff
@@ -12,34 +12,90 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  docker:
+  build-ccs-base:
     runs-on: ubuntu-latest
     permissions:
       packages: write
+
     steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3.5.3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2.9.0
+
+      - name: Login to GitHub Docker Registry
+        uses: docker/login-action@v2.2.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Get Release Tag
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> "$GITHUB_ENV"
+        run: echo "RELEASE_VERSION=v11.0.0" >> "$GITHUB_ENV"
+
+      - name: Build Base Image
+        id: docker_build_base
+        uses: docker/build-push-action@v4.1.1
+        with:
+          context: .
+          push: true
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          target: install-ccs
+          tags: >
+            ghcr.io/${{
+            github.repository_owner 
+            }}/ccs-base:${{ env.RELEASE_VERSION }}
+
+  build-cgt-images:
+    runs-on: ubuntu-latest
+    needs: build-ccs-base
+    permissions:
+      packages: write
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix:
+        cgt-versions:
+          - prefix: 18.1.3
+            version-number: 18.1.3.LTS
+            installer-url: 18.1.3.LTS/ti_cgt_msp430_18.1.3.LTS_linux_installer_x86.bin
+          - prefix: 21.6.0
+            version-number: 21.6.0.LTS
+            installer-url: 21.6.0.LTS/ti_cgt_msp430_21.6.0.LTS_linux-x64_installer.bin
+            
+    steps:
+      - name: Get Release Tag
+        # run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> "$GITHUB_ENV"
```

Comment:
can this be removed?

Reply:
Not quite, I actually needed to re-enable that line! Good catch though as I'd missed this.